server/controllers/dashboardController.js

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable max-len */
+/* eslint-disable no-console */
+import moment from "moment";
+import * as util from "util";
+import models from "../models";
+import { Op } from "sequelize";
+import paginationMeta from "../helpers/paginationHelper";
+import {paginationResponse} from "../utils/formatter";
+import {validFormat} from '../helpers/dateHelpers';
+
+
+const automation = models.Automation;
+/**
+ * Returns dateFunction
```

Comment:
Here, it might be better to say: `Returns date duration object`

Reply:
Done